src/vaadin-date-picker-mixin.html

```diff
@@ -341,7 +346,10 @@
       this.addEventListener('keydown', this._onKeydown.bind(this));
       this.addEventListener('input', this._onUserInput.bind(this));
       this.addEventListener('focus', e => this._noInput && e.target.blur());
-      this.addEventListener('blur', e => !this.opened && this.validate());
+      this.addEventListener('blur', e => {
+        this.autoOpenDisabled && this._selectAndValidateInput();
```

Comment:
This change results in a regression: it causes `validate` to be called twice in case when date picker is closed: first time in `_selectAndValidateInput` and the second time in the next string.

Please add a test that would check that `validate` is only called once.

Reply:
Should be fixed and added a test.